src/main.rs

```diff
@@ -90,6 +90,52 @@ async fn main() -> Result<(), ()> {
         }
     }
 
+    //    On startup error out if the following checks don't pass:
+    // - That GetVersion has peersrpc, signerrpc and dev build tags.
+    // - That the version is > v0.16.0-beta
+    let version_info = client
+        .versioner()
+        .get_version(VersionRequest {})
+        .await
+        .expect("failed to get version")
+        .into_inner();
+    let version = &version_info.version;
+    let tags = &version_info.build_tags;
+
+    let min_version = String::from("0.16.0-beta");
```

Comment:
We're going to recommend folks run a patch based off 16.2 (see #44), so let's bump this up.